yarn-project/noir-protocol-circuits/src/index.ts

```diff
@@ -247,7 +247,7 @@ async function executePrivateKernelInitWithACVM(input: InitInputType): Promise<R
   );
 
   // Decode the witness map into two fields, the return values and the inputs
-  const decodedInputs: DecodedInputs = abiDecode(PrivateKernelInitSimulatedJson.abi, _witnessMap);
+  const decodedInputs: DecodedInputs = abiDecode(PrivateKernelInitSimulatedJson.abi as Abi, _witnessMap);
 
   // Cast the inputs as the return type
   return decodedInputs.return_value as ReturnType;
```

Comment:
not getting why this one has typecasts?

Reply:
I don't know why it wasn't an issue before.
This is the fact that JSON doesnt have type info so string != 'blah'